CMakeLists.txt

```diff
@@ -23,33 +23,41 @@ set( ECBUILD_DEFAULT_BUILD_TYPE Release )
 
 # Enable MPI
 set( ENABLE_MPI ON CACHE BOOL "Compile with MPI")
-
-# Initialize
-ecbuild_bundle_initialize()
```

Comment:
I don't know what `ecbuild_bundle_initialize()` is doing; is it OK to remove it?

Reply:
I have no idea what it does, but I'll give it a try and report back.

Reply:
Apparently it wasn't needed. If that's the case, then we should also remove the `ecbuild_bundle_finalize()` at the end of the file?

Reply:
As long as we don't hear later that ecbuild is not behaving as intended.